packages/plugin-ext/src/main/node/plugin-deployer-entry-impl.ts

```diff
@@ -47,6 +47,7 @@ export class PluginDeployerEntryImpl implements PluginDeployerEntry {
         } else {
             this.resolved = false;
         }
+        this.storeValue('isUnderDevelopment', originId === 'Hosted Plugin');
```

Comment:
I don't think this needs to be done in the `PluginDeployerEntryImpl`. The `HostedPluginReader`(packages/plugin-dev/src/node/hosted-plugin-reader.ts) can call `storeValue` on the plugin deployer entry that it creates.